### ISSUE TITLE
Very poor results on Librispeech 100-hour dataset with mtlalpha=0 (pure attention)

### ISSUE THREAD

**Issue by kushagra1729** [espnet/espnet#1327]
Hey all, 

Thanks a lot for this wonderful repository. I was trying to train a pure attention based model (mtlalpha=0) on Librispeech 100-hours dataset. I used the default hyperparameters as given in the repository for training using an encoder-decoder architecture i.e. 5 encoder layers of 1024 units, 2 decoder layers of 1024 units just with the change that mtlalpha=ctc_weight=0. But I only got 35% training accuracy which is way too low. I have attached the training plot. I also tried a variety of other set of parameters but couldn't find a single configuration which gives decent results.

![espnet](https://user-images.githubusercontent.com/35064854/68067180-4b67d480-fd69-11e9-9b8d-bbbfd328f80c.png)

It would be great if you could suggest the correct config parameters to get decent results with pure attention on Librispeech 100-hour corpus.

Thanks!

**Comment by Cescfangs** [comment]
same problem here with 120h Mandarin data, I was expecting Espnet to **overfit** on this small dataset, but got poor results instead

![acc](https://user-images.githubusercontent.com/11382612/68095939-60b73d00-fee8-11e9-857e-4d836840b7cd.png)

**Comment by sw005320** [comment]
Is that based on transformer or RNN?

**Comment by kushagra1729** [comment]
It's based on RNN

**Comment by sw005320** [comment]
OK, could you share your training config parameters?
Could you start from the shallow models? You may also play with the scheduled sampling a bit.
I think we could make it work with some tweaking of some hyper-parameters.

**Comment by tomkocse** [comment]
I am having the same problem with librispeech 100-hour. I think the author has suggested a potential solution here:
https://github.com/espnet/espnet/issues/1318
though I don't think the problem is entirely solved.
